doc/nrf/libraries/networking/ot_rpc.rst

```diff
@@ -203,5 +216,12 @@ The libraries have the following dependencies:
 API documentation
 *****************
 
-This library does not define a new OpenThread API.
+This library mostly reimplements the existing OpenThread C API.
```

Comment:
```suggestion
This library is an implementation of the existing OpenThread C API.
```
I think this reads a bit better. "Reimplements" sounds a bit like we are disabling and enabling the OpenThread C API instead of having our own version of it.